cal-itp/calitp.org
# Customer stories: Stories page and homepage update

closes #354 

- [x] Nav: Add customer stories to nav
- [x] Stories page: Add all content and static TOC
- [x] Stories page: Impact section - Mobile
- [x] Stories page: Impact section - Large/Medium: Outer alignment
- [x] Stories page: Large/Medium: Fix the footer
- [x] Stories page: Impact section - Large/Medium: Inner alignment
- [x] Stories page: Figure out the fade in / out effect
- [x] Stories page: Add footnote links
- [x] Stories page: Anchor links
- [x] Home page: Add stats
- [x] Stories/Partners page: Footnote code

## How to test this PR
- Enter the page through an anchor tag: https://deploy-preview-380--cal-itp-website.netlify.app/customer-stories#stories-GTFS-VCTC https://deploy-preview-380--cal-itp-website.netlify.app/customer-stories#stories-benefits-MST
- Enter without anchor tag https://deploy-preview-380--cal-itp-website.netlify.app/customer-stories
- Click on the sidebar links - active area should be in blue
- After clicking on a sidebar link, click Back on the browser. The URL should change and the active title should match the browser URL.
- Check the stats on the homepage: https://deploy-preview-380--cal-itp-website.netlify.app
- Nav should have `Customer stories` highlighted

## Sidebar
- [x] Desktop: The sidebar stays sticky on scroll, but also goes behind the Impact section (and has the fade background)
- [x] Clicking / tabbing and entering on a link takes you to the section
- [x] Clicking / tabbing on a single link will make _both_ pairs of links colored in styling
- [x] If you enter the page with an anchor tag, like `#stories-GTFS-VCTD`, then the link pairs will be set to active and styled appropriately
